modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlAsynchronousApiTest.java

```diff
@@ -326,7 +326,7 @@ public void checkTransactionsWithDml() throws Exception {
 
         checkDml(2 * ROW_COUNT, ses, "DELETE FROM TEST WHERE VAL0 >= 0");
 
-        assertEquals(ROW_COUNT + 1 + 1 + 1 + 1, txManagerInternal.finished() - txPrevCnt);
+        assertEquals(ROW_COUNT + 1 + 1 + 1 + 1 + 1, txManagerInternal.finished() - txPrevCnt);
```

Comment:
Could you please name all 5 transactions?

Reply:
All those transactions were here before my change.
Another one is a transaction which appears for SQL SELECT (it was not considered before, because the scan was not transactional)